app/views/service-start/index.njk

```diff
@@ -17,6 +17,8 @@
       <li>Farming Equipment and Technology Fund</li>
     </ul>
 
+    <p>You can also <a id='viewYearlyTotals' href="/viewSchemePaymentsByYear">view yearly totals.</a></p>
+
     {{ govukInsetText({
       html: "This service shows payments made between 1 April 2021 and 31 March 2022."
```

Comment:
Why is this hard coded to only 2 years? Are we not supporting up to 5 years data?

Reply:
That hasn't changed in this branch. We have not yet done the work to update the service start page to bring it up-to-date with scalable view yet. That will be in a separate ticket.

Reply:
ok